compiler/rustc_middle/src/hir/mod.rs

```diff
@@ -53,19 +41,19 @@ impl<'a, 'tcx> HashStable<StableHashingContext<'a>> for Owner<'tcx> {
 /// The parent is trash when the node is a HIR owner.
 #[derive(Clone, Debug)]
 pub struct ParentedNode<'tcx> {
-    parent: ItemLocalId,
-    node: Node<'tcx>,
+    pub parent: ItemLocalId,
+    pub node: Node<'tcx>,
 }
 
 #[derive(Debug)]
 pub struct OwnerNodes<'tcx> {
```

Comment:
I find the name of this struct a bit confusing. I read it as "a set of `OwnerNode` items" but it is really "all the node related information for a single Owner", right? Could this whole struct be merged into `OwnerInfo`?